Products/ZenModel/tests/testDevice.py

```diff
@@ -231,13 +258,6 @@ def testSetManageIp(self):
         self.dev.setManageIp(testIp)
         self.assertEqual(self.dev.getManageIp(), testIp)
 
-        # Need a network interface to register an IP in catalog
-        from Products.ZenModel.IpInterface import IpInterface
-        tmpIface = IpInterface('testNIC')
-        self.dev.os.interfaces._setObject('testNIC', tmpIface)
-        self.iface1 = self.dev.getDeviceComponents()[0]
-        self.iface1.addIpAddress('1.2.3.4')
```

Comment:
I removed it from here because before my changes it was necessary to create an Interface with Ip Addr to emulate modeled device behavior since afterward, we were using that IP for checking duplicate devices. Now that Ip address is created when the device is created with manageIp even if there is no Interface on it.

But this testcase with addIpAddress is valid, so I will put it back, to check case when device firstly create an Ip and then Interface also try to assign that ip to itself.